client/components/web-preview/style.scss

```diff
@@ -34,6 +34,7 @@
 		right: 0;
 		bottom: 0;
 	display: block;
+	background: fade-out( $gray-light, .2 );
```

Comment:
I prefer to use `rgba( )` here because it's a bit more standard in Calypso. I didn't know what fade-out did. Apparently we use it in a few places.